_pages/arifcatalyst.yaml

```diff
@@ -0,0 +1,5 @@
+---
+githubHandle:arifcatalyst
+pageUrl:arifcatalyst.github.io/Core-Java
+timestamp: 2017-03-29
+---

```

Comment:
@arifcatalyst you'll also need a space after each `:`. What text editor are you using? It seems to be adding a `^M` character.